ash/src/device.rs

```diff
@@ -2,9 +2,15 @@
 use crate::prelude::*;
 use crate::vk;
 use crate::RawPtr;
-use std::mem;
-use std::os::raw::c_void;
-use std::ptr;
+use core::mem;
+use core::ptr;
+
+#[cfg(not(feature = "std"))]
+use alloc::vec::Vec;
+#[cfg(not(feature = "std"))]
+use core::ffi::{c_void, CStr};
+#[cfg(feature = "std")]
+use std::ffi::{c_void, CStr};
```

Comment:
Can you document with a comment that this nastiness is only required prior to 1.64 (`std::ffi::` is just a reexport on `core::ffi::` after), that way we can remove it when MSRV is inevitably bumped at some point down the road.
(Everywhere, not just in the generator)

FWIW I wouldn't mind doing this _once_ in a prelude (existing or new module) and importing from that.

Reply:
As requested, can you document this? Once we bump MSRV for the non-`no_std` case to 1.64 too we can just get rid of the `std::ffi` import altogether and unconditionally use `std::ffi`.